source/apps/copy.txt

```diff
@@ -0,0 +1,200 @@
+.. _copy-app:
+
+===========
+Copy an App
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+You can make a copy of an existing App by reusing the App's
+configuration files and manually porting other information like Secrets.
+
+You might want to copy an application if:
+
+- You use feature branches for development. Use a unique copy of the
+  App for each feature branch to avoid conflicts.
+
+- You run tests against a working version of the App. You can copy an
+  App for each test run to to ensure a consistent start state.
+
+- You deploy the same app to clusters across regions using a :ref:`local
+  deployment model <local-deployment>`. You can copy an App to servce
+  multiple regions locally.
+
+Before You Begin
+----------------
+
+.. tabs-realm-admin-interfaces::
+
+   .. tab::
+      :tabid: cli
+
+      You will need the following to copy an App in the CLI:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+
+      - .. include:: /includes/prereqs/atlas-admin-api-key.rst
+
+      - .. include:: /includes/prereqs/app-services-cli.rst
```

Comment:
not part of this PR so can't touch but this sentence reads funky to me:

> A [properly installed](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-apps-copy/cli/#std-label-install-realm-cli) copy of [realm-cli](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-apps-copy/cli/#mongodb-binary-bin.realm-cli) that has been added to your system PATH.

perhaps refactor to smthn like: 

"realm-cli installed and added to your system PATH."